static/js/interpreter.js

```diff
@@ -119,7 +119,7 @@ export function initializeInterpreter(divName) {
         editor.commands.addCommand({
             name: "ctrl-enter",
             bindKey: { win: "Ctrl+Enter", mac: "Cmd+Enter"},
-            exec: function(editor, ...rest) {run(editor.getValue().replace(/\r/g, ""), db);}
```

Comment:
What does this have to do with aggregation

Reply:
uhh it's also a bug that I noticed while testing an aggregation?